projects/js-packages/charts/src/providers/theme/themes.ts

```diff
@@ -0,0 +1,48 @@
+import type { ChartTheme } from '../../components/shared/types';
+
+/**
+ * Default theme configuration
+ */
+const defaultTheme: ChartTheme = {
+	backgroundColor: '#FFFFFF',
+	colors: [ '#98C8DF', '#006DAB', '#A6DC80', '#1F9828', '#FF8C8F' ],
+	gridStyles: {
+		stroke: '#787C82',
+		strokeWidth: 1,
+	},
+	tickLength: 0,
+	gridColor: '',
+	gridColorDark: '',
+};
+
+/**
+ * Jetpack theme configuration
+ */
+const jetpackTheme: ChartTheme = {
+	backgroundColor: '#FFFFFF',
+	colors: [ '#98C8DF', '#006DAB', '#A6DC80', '#1F9828', '#FF8C8F' ],
```

Comment:
Can we import color vars from `color-studio` for these themes, or would that be adding unnecessary extra dependencies?

Reply:
Good point. I'll add a task to check this and see how/if we can leverage CSS variables for it.